packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -817,6 +824,508 @@ let rows = [
+*/
+.aria-table {
+  border-collapse: collapse;
+  width: 300px;
+  height: 200px;
+  display: block;
+  position: relative;
+  overflow: auto;
+}
+
+
+.aria-table-rowGroupHeader {
+  border-bottom: 2px solid var(--spectrum-global-color-gray-800);
+  position: sticky;
+  top: 0;
+  background: var(--spectrum-gray-100);
+  width: fit-content;
+}
+
+.aria-table-rowGroupBody {
+  max-height: 200px;
+}
+```
+</details>
+
+### Resizable table header
+
+The `TableRowGroup` and `TableHeaderRow` only require minor style changes to accommodate the new `display` changes made in
+`ResizableColumnsTable`.
+
+```tsx example export=true render=false
+function ResizableTableRowGroup({type: Element, className, children}) {
+  let {rowGroupProps} = useTableRowGroup();
+  return (
+    <Element
+      /*- begin highlight -*/
+      className={`aria-table-rowGroup ${className}`}
+      /*- end highlight -*/
+      {...rowGroupProps}>
+      {children}
+    </Element>
+  );
+}
+```
+
+```tsx example export=true render=false
+function ResizableTableHeaderRow({item, state, children}) {
+  let ref = useRef();
+  let {rowProps} = useTableHeaderRow({node: item}, state, ref);
+
+  return (
+    <tr
+      {...rowProps}
+      /*- begin highlight -*/
+      className="aria-table-row"
+      /*- end highlight -*/
+      ref={ref}>
+      {children}
+    </tr>
+  );
+}
+```
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+
+```css
+.aria-table-rowGroup {
+  display: block;
+}
+
+.aria-table-row {
+  display: flex;
+}
+```
+</details>
+
+The `TableColumnHeader` is where we see the bulk of the changes required to support resizable columns. First of all, we need to accommodate a `Resizer` element in every resizable column that
+the user can drag or focus to perform a resize operation. Since the resizer will be a focusable element within the table header, we need to make the header title a focusable element as well so keyboard
+focus won't be immediately sent to the resizer as you navigate between the column headers. Finally, we apply the computed width of our column from <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTableColumnResizeState} />
+to the header element.
+
+```tsx example export=true render=false
+// Reuse the Button from your component library. See below for details.
+import {Button} from 'your-component-library';
+
+function ResizableTableColumnHeader({column, state, layoutState, onResizeStart, onResize, onResizeEnd}) {
+  let {widths} = layoutState;
+  let allowsResizing = column.props.allowsResizing;
+  let ref = useRef(null);
+  let {columnHeaderProps} = useTableColumnHeader({node: column}, state, ref);
+
+  return (
+    <th
+      {...columnHeaderProps}
+      className="aria-table-headerCell"
+      style={{width: widths.get(column.key)}}
+      ref={ref}>
+      <div style={{display: 'flex', position: 'relative'}}>
+        <Button className="aria-table-headerTitle">
```

Comment:
It needs to be some kind of focusable element since focus will move to the [first focusable child](https://github.com/adobe/react-spectrum/blob/a16e8474ed620c8898b1c8133ad5ec796b25a773/packages/%40react-aria/table/src/useTableColumnHeader.ts#L48) due to `focusMode: child`. If I don't make the header content focusable, then focus will jump to the resizer. 

A button does feel odd since there isn't a press action but the other focusable elements don't feel appropriate either